service/amqp/src/main/scripts/org/kaazing/gateway/service/amqp/amqp091/ws.accept.rpt

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
Any reason why you can't just use ws.accept.rpt script from your new k3po spec project?